crates/rspack_plugin_css/src/utils.rs

```diff
@@ -325,7 +325,9 @@ static TRIM_WHITE_SPACES: LazyLock<Regex> =
 static UNESCAPE: LazyLock<Regex> =
   LazyLock::new(|| Regex::new(r"\\([0-9a-fA-F]{1,6}[ \t\n\r\f]?|[\s\S])").expect("Invalid RegExp"));
 
-static DATA: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"^(?i)data:").expect("Invalid RegExp"));
+fn is_data_uri(s: &str) -> bool {
+  s.to_lowercase().starts_with("data:")
```

Comment:
Please avoid redundant memory allocation, you can use `str::eq_ignore_ascii_case`